tike/tomo.py

```diff
@@ -182,77 +142,23 @@ def reconstruct(
+                      **kwargs,
                       )
     Li = tomopy.recon(tomo=line_integrals.imag,
                       theta=theta,
-                      algorithm=algorithm,
                       init_recon=obj.imag,
-                      num_iter=niter, **kwargs,
+                      **kwargs,
                       )
     recon = np.empty(Lr.shape, dtype=complex)
     recon.real = Lr
     recon.imag = Li
     return recon
-    # obj, obj_corner, probe, theta, v, h \
-    #     = _tomo_interface(obj, obj_corner, probe, theta, v, h)
-    # assert niter >= 0, "Number of iterations should be >= 0"
-    # # Send data to c function
-    # logger.info("{} on {:,d} element grid for {:,d} iterations".format(
-    #             algorithm, obj.size, niter))
-    # ngrid = obj.shape
-    # line_integrals = utils.as_float32(line_integrals)
-    # theta = utils.as_float32(theta)
-    # v = utils.as_float32(v)
-    # h = utils.as_float32(h)
-    # obj = utils.as_float32(obj)
-    # # Add new tomography algorithms here
-    # # TODO: The size of this function may be reduced further if all recon
-    # #   clibs have a standard interface. Perhaps pass unique params to a
-    # #   generic struct or array.
-    # if algorithm is "art":
-    #     LIBTIKE.art.restype = utils.as_c_void_p()
-    #     LIBTIKE.art(
-    #         utils.as_c_float(obj_corner[0]),
-    #         utils.as_c_float(obj_corner[1]),
-    #         utils.as_c_float(obj_corner[2]),
-    #         utils.as_c_int(ngrid[0]),
-    #         utils.as_c_int(ngrid[1]),
-    #         utils.as_c_int(ngrid[2]),
-    #         utils.as_c_float_p(line_integrals),
-    #         utils.as_c_float_p(theta),
-    #         utils.as_c_float_p(h),
-    #         utils.as_c_float_p(v),
-    #         utils.as_c_int(line_integrals.size),
-    #         utils.as_c_float_p(obj),
-    #         utils.as_c_int(niter))
-    # elif algorithm is "sirt":
-    #     LIBTIKE.sirt.restype = utils.as_c_void_p()
-    #     LIBTIKE.sirt(
-    #         utils.as_c_float(obj_corner[0]),
-    #         utils.as_c_float(obj_corner[1]),
-    #         utils.as_c_float(obj_corner[2]),
-    #         utils.as_c_int(ngrid[0]),
-    #         utils.as_c_int(ngrid[1]),
-    #         utils.as_c_int(ngrid[2]),
-    #         utils.as_c_float_p(line_integrals),
-    #         utils.as_c_float_p(theta),
-    #         utils.as_c_float_p(h),
-    #         utils.as_c_float_p(v),
-    #         utils.as_c_int(line_integrals.size),
-    #         utils.as_c_float_p(obj),
-    #         utils.as_c_int(niter))
-    # else:
-    #     raise ValueError("The {} algorithm is not an available.".format(
-    #         algorithm))
-    # return obj
 
 
 def forward(
-        obj=None, obj_corner=None,
-        probe=None, theta=None, v=None, h=None,
+        obj=None,
+        theta=None,
         **kwargs
 ):
     """Compute line integrals over an obj; i.e. simulate data acquisition."""
```

Comment:
Saying this is for simulating data acquisition can be confusion, because it is not. I'd say i.e. projections of an object for each rotation angle.